### PR TITLE
test(module): increase coverage

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
Added more tests

### CODE REVIEW

File: test/axios.test.js [hunk 1]
```diff
@@ -80,15 +58,117 @@ describe('axios module', () => {
...
+describe('module', () => {
+  beforeAll(async () => {
+    nuxt = new Nuxt(config)
+
+    // Spy addTemplate
+    addTemplate = nuxt.moduleContainer.addTemplate = jest.fn(
+      nuxt.moduleContainer.addTemplate
+    )
+
+    await new Builder(nuxt).build()
```
Comment (pi0) [inline test/axios.test.js]: This `build()` call is costly for each test and unnecessary for just coverage. I suggest using `generateRouteAndFiles`. Have to look into the nuxt code to point to an example.

Comment (ricardogobbosouza) [reply test/axios.test.js]: @pi0 I think it got better and faster

File: test/axios.test.js [hunk 2]
```diff
@@ -2,39 +2,34 @@ jest.setTimeout(60000)
...
-    await new Builder(nuxt).build()
-    await nuxt.listen(3000)
-  })
+  const build = new Builder(nuxt)
 
-  afterAll(async () => {
-    await nuxt.close()
-  })
+  await build.validatePages()
+  await build.generateRoutesAndFiles()
```
Comment (pi0) [inline test/axios.test.js]: Nice!

**Comment by pi0** [comment]
I've enabled CI but seems not working for this PR.

**Comment by ricardogobbosouza** [comment]
@pi0 need re-create webhook for circleci

**Comment by ricardogobbosouza** [comment]
@pi0 codecov should be enabled too?

**Pull request merged**